setup.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages, setup
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.md') as history_file:
+    history = history_file.read()
+
+install_requires = [
+    'certifi==2019.3.9',
```

Comment:
Done. This had been inherited from the old `requirements.txt` and was not reviewed.
I removed several libraries that were not used and specified safe ranges instead of fixed versions.